### ISSUE TITLE
Any recommended decks other than the 3 on Readme?

### ISSUE THREAD

**Issue by nezam7** [MOj0/LoR-Bot#21]
Not an issue. More like a discussion. Just want to know what decks you guys are using. 
Any recommended standard deck?

Also.. maybe not be a bad idea if we can have a discord channel to discuss the deck, an improvement on the bot, etc.

Thanks for making this bot works. Helps a ton

**Comment by MOj0** [comment]
Yo,
in general the all-in aggro decks should work the best, since that is the only thing the bot is doing :smile:.

So, development of this bot is kind of stagnated. The thing is, to properly make such a bot, the best way is to read the memory of the game directly and do actions based of that, which I'm not doing (also that is easier to detect and ban). So since this bot is made in such a (primitive) way, it is very hard to build on top of it, if you get what I'm saying :) 

I can make a simple discord server if you want and I can do simple bug fixes, but not huge improvements I'm afraid.

**Comment by nezam7** [comment]
i want to make a standard deck aggro. do u have any good deck for the bot? (standard)

**Comment by MOj0** [comment]
Hi, well I put together this shurima aggro deck. 
CMCQCAYGBYAQIBQNAECQMAIEAQDQGGRGGMDAEBQICQLC2MB4AAAQCAQGFA
It's not really amazing, perhaps I would need to add a separate logic just for that deck (just like with pirates and ephemeral decks).

**Comment by nezam7** [comment]
i mesh up some random aggro deck i found on mobalytics the other day but somehow the bot didnt run. i got back into ephemereal and pirate decks. will try this for sure. thanks

**Comment by MOj0** [comment]
Yeah, when making a deck, you must take into account that bot doesn't know how to play all the cards. In general, it doesn't know how to do targeting, except for something like Oblivious Islander, which targets the cards in hand.
That being said, if you create a custom strategy (like pirates and ephemeral), you can code that logic in as well.

**Comment by nezam7** [comment]
![image](https://github.com/MOj0/LoR-Bot/assets/4648255/ab340451-2eb2-42fd-9031-a312e205384b)

managed to reached master with 100% bot. i use same modified aggro deck. i think at this point its probably nice to have a discord to talk about deck strategies like what are the optimum w/r with standard/wild deck etc.

**Comment by MOj0** [comment]
Oh wow, I didn't think the bot would be this capable, to be honest. If you could share the deck code and your match history, so I can put it in the readme, that would be awesome :D
This is my discord username if you want to chat/talk: mojo7683

**Comment by nezam7** [comment]
i have added you. name nezam3035

**Comment by MOj0** [comment]
I didn't get anything, and I also could not add you, says username is not found...

**Comment by nezam7** [comment]
My username is just Nezam

I think its because Discord already made changes on username policy recently

**Comment by MOj0** [comment]
Right, but it should still work as my username is mojo7683...
Anyways I sent you a friend request and it went through.

**Comment by LunaCasey** [comment]
I made it to Masters in Eternal with a modified Pirate deck.
Here's the deck: CICACAQDAQAQKBQBAUAQGAQMEUTCQBICAYEBMJRNHQAQEAIDA4NAA

I also wanted to ask, how do I add a custom strategy?

**Comment by MOj0** [comment]
Hi @LunaCasey, thanks for your feedback and the deck code, I'll update the readme with this.

Regarding a custom strategy, there is a `Strategy` class [here ](https://github.com/MOj0/LoR-Bot/blob/master/Strategy.py). To add a custom strategy, you need to add another class, which extends `Strategy` class, like how it's done in `Ephemeral` [for example](https://github.com/MOj0/LoR-Bot/blob/master/Ephemeral.py). Then you need to add it to the `DeckType` enum [here](https://github.com/MOj0/LoR-Bot/blob/master/StateMachine.py#L17). Lastly, you need to add some logic, by which it will select that deck type [here](https://github.com/MOj0/LoR-Bot/blob/master/StateMachine.py#L202C11-L211). If you have any trouble, comment here, or make a new Issue... perhaps I'll need to add this procedure to the readme aswell...

**Comment by ajdc2** [comment]
Has anyone had any success with the bot since the new expansion ? Now that ranked is standard the kalista deck seems gone and pirates lost some good cards. I have spent the past weeks messing around with pirates trying to find replacements but my best attempts don't exceed like 20% winrate. Has anyone had more luck ?
